components/micro-gateway-core/src/main/ballerina/src/gateway/handler_providers/jwt_auth_provider.bal

```diff
@@ -141,3 +147,62 @@ public function validateSubscriptions(string jwtToken, jwt:JwtPayload payload, b
     setErrorMessageToInvocationContext(API_AUTH_FORBIDDEN);
     return prepareError("Failed to decode the JWT.");
 }
+
+public function doMappingContext(runtime:InvocationContext invocationContext,jwt:JwtPayload payload) returns @tainted (auth:Error)? {
+    string payloadissuer=payload["iss"].toString();
+    map<any>? customClaims = invocationContext["principal"]["claims"];
+    map<any>? invocationvalue = invocationContext;
+    map<anydata>[] | error jwtIssuers = map<anydata>[].constructFrom(config:getAsArray(JWT_INSTANCE_ID));
```

Comment:
Why are we reading the jwt issuers again from config? Can we read custom claims in JWT handler creation time without reading the config again here?